src/Mailer/ValidateUsernameEmail.php

```diff
@@ -48,4 +63,14 @@ public function send(UserInterface $user): \Swift_Message
             ]
         );
     }
+
+    private function getRoute(): string
```

Comment:
Not really DRY code, the same function as in RegistrationEmail

Reply:
True, but since it's merely retrieving the route from the request, I think it does not justify restructuring the entire mailing part.